lib/vrl/stdlib/src/compact.rs

```diff
@@ -75,7 +75,7 @@ impl Function for Compact {
         let recursive = arguments.optional("recursive");
         let null = arguments.optional("null");
         let string = arguments.optional("string");
-        let map = arguments.optional("map");
+        let  object = arguments.optional("object");
```

Comment:
Extra space?

Reply:
Oh, yeah, weird, `rustfmt` didn't catch that.